### DISCUSSION TITLE
[Bug Report] ShopReceipt’s formatted_address field no longer available

### DISCUSSION THREAD

**Discussion by schang969** [etsy/open-api#1247] [Q&A]
I'm using a personal key from Taiwan, and I understand formatted_address is no longer available in Turkey.

How do I make a request to the API to re-enable the formatted_address field? Thanks!

**Comment by schang969** [comment]
Follow up to my previous post:

- I'm now still able to get buyer name and individual address lines for my open orders from US buyers
- But no address info is available for my open orders from EU buyers

Can someone from Etsy please give an update? Thank you.

**Comment by schang969** [comment]
NOTE Access to ShopReceipt's first_line, second_line, city, state, zip, country_iso and formatted_address is contingent in some regions to a preferred partnership status with Etsy

------------------------------------------------

Etsy can you at least clarify the "regions"??  Why can't I have access to the shipping address of open orders in MY own shop??

**Reply by emergingdzns** [reply]
@schang969 For the most part, Etsy staff don't monitor this board. Anymore, it's meant for discussion between devs and helping each other. You need to report this to developer@etsy.com.

**Reply by roborock** [reply]
any news ? im stuck in same problem 

some shop receipts doesnt fill address lines.

**Reply by jack-ditto** [reply]
Anyone find a solution to this? I have reached out to developer@etsy.com but no response yet.
